packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.tsx

```diff
@@ -121,14 +129,13 @@ const PanelView: React.FC<PanelViewProps> = ({ expanded, testUuid, basePath, typ
               totalSearchResults={filteredPanels?.length ?? 0}
             />
             {!isLoading ? (
-              panels?.length > 0 ? (
```

Comment:
This return block (see folded code above) is for non-tablet viewers. But the only usages of PanelView are tablet and print. So I'm not sure whether or not this is used at all. I didn't touch the print stuff.